install_files/ansible-base/roles/build-securedrop-app-code-deb-pkg/files/rsync-filter

```diff
@@ -0,0 +1,36 @@
+exclude config.py
+include *.py
+include COPYING
+include dictionaries/
+include dictionaries/*.txt
+include journalist_templates/
+include journalist_templates/*.html
+include management/
+include management/*.py
+include requirements/
+include requirements/securedrop-requirements.txt
+include source_templates/
+include source_templates/*.html
+include static/
+include static/fonts/
+include static/fonts/**
+include static/i/
+include static/i/**
+include static/i/font-awesome/
+include static/i/font-awesome/**
+include static/i/tipbox/
```

Comment:
Is this a really old legacy folder @conorsch? Not present in any of our active branches afaicr.